pkg/sink/sink.go

```diff
@@ -554,7 +558,24 @@ func (r Sink) ExecuteInterceptors(trInt []*triggersv1.TriggerInterceptor, in *ht
 		}
 		// Clear interceptorParams for the next interceptor in chain
 		request.InterceptorParams = map[string]interface{}{}
+
+		// check if string is urlencoded
```

Comment:
The logic here looks fine but shouldn't we call it when we first create the request object around line 467?

Reply:
@dibyom, good call. It simplified the Slack interceptor. 
All tests are ok on my local but for some reason [this](https://github.com/ilan-pinto/triggers/blob/79a1e8fd69901eb2baaf37b3f00f5b828e979c54/test/eventlistener_test.go#L127) test fails in GitHub action